src/client/Microsoft.Identity.Client/AppConfig/ExternalTokenResult.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Token result from external token provider
+    /// </summary>
+    public class ExternalTokenResult
+    {
+        /// <summary>
+        /// Correlationn Id to track request
+        /// </summary>
+        public string CorrelationId { get; set; }
+
+        /// <summary>
+        /// Tenant Id for client application
+        /// </summary>
+        public string TenantId { get; set; }
+
+        /// <summary>
+        /// Cancellation token for async operation
+        /// </summary>
+        public CancellationToken CancellationToken { get; set; }
```

Comment:
Not needed, you don't use this.

Reply:
was in the spec so wasn't sure if was going to be used for something down the road. will remove